docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     image: postgres
     environment:
       PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_PASSWORD: $OSEM_DB_PASSWORD
```

Comment:
Since we are using postgres image, it seems that we need this. Wondering why we didn't have that before...

Reply:
It's a recent change that caught many people by surprise; discussion in docker-library/postgres#681.